di-ipv-evcs-stub/lambdas/src/services/evcsService.ts

```diff
@@ -0,0 +1,97 @@
+import { DynamoDB, GetItemInput, PutItemInput } from "@aws-sdk/client-dynamodb";
+import { marshall, unmarshall } from "@aws-sdk/util-dynamodb";
+
+import PostRequest from "../domain/postRequest";
+import VcStateMetadata from "../domain/vcStateMetadata";
+import ServiceResponse from "../domain/serviceResponse";
+import AllState from "../domain/enums/allState";
+import EvcsVcItem from "../model/evcsVcItem";
+
+import { config } from "../common/config";
+import { getSsmParameter } from "../common/ssmParameter";
+import { v4 as uuid } from "uuid";
+
+const dynamoClient = config.isLocalDev
+  ? new DynamoDB({
+      endpoint: config.localDynamoDbEndpoint,
+      region: config.region,
+    })
+  : new DynamoDB({ region: config.region });
+
+export async function processPostUserVCsRequest(
+    userId: string,
+    postRequest: PostRequest
+  ): Promise<ServiceResponse> {
+  console.info(`Post user record.`);
+  for (const persistVC of postRequest.persistVCs) {
+      const vcItem: EvcsVcItem = {
+          userId: userId,
+          vc: persistVC.vc,
+          state: persistVC.state,
+          metadata: persistVC.metadata!,
+          provenance: persistVC.provenance!,
+          ttl: await getTtl(),
+        };
+        await saveUserVC(vcItem);
+  }
+
+  return {
+    response: {
+      messageId: uuid()
+    },
+    statusCode: 202
+  };
+}
+
+export async function processGetUserVCsRequest(
+  userId: string
+): Promise<ServiceResponse> {
+  console.info(`Get user record.`);
+  const getItemInput: GetItemInput = {
+    TableName: config.evcsStubUserVCsTableName,
+    Key: marshall({
+      userId: userId,
+    }),
+  };
+  const items = (await dynamoClient.scan(getItemInput)).Items;
+  console.info(`Total VCs retrived - ${JSON.stringify(items?.length)}`);
+  const evcsVcItems: EvcsVcItem[] = [];
+  items?.forEach((item) => {
+    evcsVcItems.push(unmarshall(item) as EvcsVcItem);
+  });
```

Comment:
You can map here:
```
const items = (await dynamoClient.scan(getItemInput)).Items ?? [];

const evcsVcItems = items.map((item) => unmarshall(item) as EvcsVcItem);
```
(note that this does not actually check that `item` is the right type, but given we own this DB it should be ok)